tests/validation/cattlevalidationtest/core/test_machine.py

```diff
@@ -0,0 +1,277 @@
+from common_fixtures import *  # NOQA
+import traceback
+
+DEFAULT_TIMEOUT = 450
+
+# Digital Ocean configurations
+access_key = os.environ.get('DIGITALOCEAN_KEY')
+image_name = "ubuntu-14-10-x64"
```

Comment:
Might want to move these to a properties file. If we need a properties file framework for the validation tests, maybe a dev can help with that, as it can become a bit of a headache to do it right.